app/pulp/app/tasks/repository.py

```diff
@@ -0,0 +1,22 @@
+from celery import shared_task
+
+from pulp.app import models
+from pulp.tasking.tasks import UserFacingTask
+
+
+@shared_task(base=UserFacingTask)
+def delete_publisher(repo_name, publisher_name):
+    models.Publisher.objects.filter(name=publisher_name, repository__name=repo_name).delete()
```

Comment:
Maybe it makes sense to remove the implementation of both of these tasks from pulp2 codebase in this 3.0-dev branch. 
And also we should not forget about cleanup by plugins which happened in these tasks in pulp 2.